docs/admin/components/layout.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Layout components
```

Comment:
Is this missing in the menu?

Reply:
This will be in next version. Layout has some changes in new version.